neuralcoref/neuralcoref.pyx

```diff
@@ -892,23 +901,17 @@ cdef class NeuralCoref(object):
         return self.get_word_embedding(sent.doc[i])
 
     def get_average_embedding(self, Span span):
-        cdef int i
-        cdef int n = 0
-        embed_arr = numpy.zeros(self.static_vectors.shape[1], dtype='float32')
-        for token in span:
-            if token.lower not in PUNCTS:
```

Comment:
Why did you remove these?

Reply:
This is all about removing the call to `numpy.zeros()`. Once I had replaced it with `sum()`, the code collapsed into just those two lines. Other than the location of the output vector, it should perform exactly the same way.